supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -69,6 +69,9 @@ object Run {
       rpargs.setOutputs(jesParameters.collect({ case i: JesFileOutput => i.name -> i.toGoogleRunParameter }).toMap.asJava)
       logger.debug(s"Outputs:\n${stringifyMap(rpargs.getOutputs.asScala.toMap)}")
 
+      val labels = Map("cromwell-workflow-id" -> s"cromwell-${workflow.id.toString}")
```

Comment:
Oh great catch!  This likely needs to be the main workflow ID to satisfy the requirements of the ticket.